connectors/snmp-connector/snmpConnectorModel.go

```diff
@@ -1,15 +1,23 @@
 package main
 
 import (
+	"fmt"
 	"github.com/gwos/tcg/connectors"
 	"github.com/gwos/tcg/connectors/snmp-connector/clients"
 	"github.com/gwos/tcg/connectors/snmp-connector/utils"
 	"github.com/gwos/tcg/log"
 	"github.com/gwos/tcg/milliseconds"
 	"github.com/gwos/tcg/transit"
+	"github.com/patrickmn/go-cache"
 	"time"
 )
 
+// FiveMinutes NeDi interval in seconds
+const FiveMinutes = 300
+
+// PreviousValueCache cache to handle "Delta" metrics
+var PreviousValueCache = cache.New(-1, -1)
```

Comment:
Should this be privately named: 
var previousValueCache

Reply:
All our constant cache variables are public, because we should be able to use it in another parts of code(maybe in future). Anyway this cache is thread safe.

Reply:
So its ok to allow package external access to our internal package implementation? Shouldn't we define clear access points like public package methods?

Reply:
Done ✓

Reply:
Had an offline discussion, closing this issue